src/Documenter.jl

```diff
@@ -785,4 +804,263 @@ function getenv(regex::Regex)
     error("could not find key/iv pair.")
 end
 
-end # module
+<<<<<<< HEAD
```

Comment:
Messed up rebase

Reply:
Huh, I spotted this last night and could have sworn I pushed the fix, but I guess not.